daffodil-test/src/test/scala/org/apache/daffodil/usertests/RC-Test.scala

```diff
@@ -17,30 +17,23 @@
 
 package org.apache.daffodil.usertests
 
-import org.junit.Test
+// import org.junit.Test
 import org.apache.daffodil.tdml.Runner
 import org.junit.AfterClass
 
-object RCTest5 {
+object RCTest {
   val testDir = "/org/apache/daffodil/usertests/"
-  val runner = Runner(testDir, "test-5.tdml.xml")
   val runner6 = Runner(testDir, "test-6.tdml.xml")
 
   @AfterClass def shutDown(): Unit = {
-    runner.reset
     runner6.reset
   }
 
 }
 
-class RCTest5 {
+class RCTest {
 
-  import RCTest5._
-
-  @Test def test5p(): Unit = { runner.runOneTest("parse-test-5") }
-
-  // DAFFODIL-2217
-  // @Test def test5u() { runner.runOneTest("unparse-test-5") }
+  //import RCTest._
 
   // DAFFODIL-2219
```

Comment:
Is this test fixed as part of this PR, or is 2219 not full resolved?

Reply:
It looks like the ticket references were swapped in the source code. [DAFFODIL-2219](https://issues.apache.org/jira/browse/DAFFODIL-2219) is resolved and that was for unparse-test-5. The issue referenced in parse-test-6 is ticket [DAFFODIL-2217](https://issues.apache.org/jira/browse/DAFFODIL-2217) and is still open.